### PR TITLE
Don't build ignition packages for RHEL

### PR DESCRIPTION
**Pull request by cottsay**
We don't have ignition packages for RHEL, so we shouldn't try to build these packages. They will always fail to build.

### CODE REVIEW

File: rolling/release-rhel-build.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ package_blacklist:
 - control_box_rst  # coinor-libipopt-dev has no RPM for RHEL 8
 - gazebo_dev  # Gazebo has no RPM package
 - grbl_ros  # Not yet generated for RHEL
+- ign_rviz_common  # ignition has no RPM packages for RHEL
```
Comment (clalancette) [inline rolling/release-rhel-build.yaml:30]: Should we also add `ign_rviz` and `ign_rviz_plugins`?

File: rolling/release-rhel-build.yaml [hunk 2]
```diff
@@ -38,6 +39,8 @@ package_blacklist:
 - rc_dynamics_api  # Not yet generated for RHEL
 - rc_genicam_api  # Not yet generated for RHEL
 - rc_genicam_driver  # Not yet generated for RHEL
+- ros_ign_bridge  # ignition has no RPM packages for RHEL
+- ros_ign_gazebo  # ignition has no RPM packages for RHEL
```
Comment (clalancette) [inline rolling/release-rhel-build.yaml:43]: Similarly, should we add `ros_ign`, `ros_ign_gazebo_demos`, and `ros_ign_image`?

File: rolling/release-rhel-build.yaml (hunk 1 above)
Comment (cottsay) [reply rolling/release-rhel-build.yaml:30]: They're all downstream of the listed packages.

**Review by chapulina — APPROVED** [review]
Ouch, I didn't know this step would be necessary. Bloom warned me that RHEL wouldn't work and I ignored it as we've been doing for packages.osrfoundation.org, documented [here](https://github.com/ignitionrobotics/ros_ign/blob/melodic/RELEASING.md#releasing-into-httpspackagesosrfoundationorg).

Is there something I could have done to avoid the need for this PR?

**Comment by cottsay** [comment]
> Is there something I could have done to avoid the need for this PR?

Nope! Until we have a better way to detect this, the easiest thing to do is just wait for the jobs to fail and then exclude them.

Thanks for reviewing the change.

**Pull request merged**